### ISSUE TITLE
CONCERN: Health Authority UX concerns - DHM-S1-20

### ISSUE THREAD

**Issue by diarmidmackenzie** [Path-Check/covid-safe-paths#457]
Several UX aspects that I found unintuitive [DHM-S1-20]
o   “Save” (floppy disk) icon used to add an entry.  This doesn’t match any UX mechanic that I am familiar with in mobile apps for adding entries to a list.
o   No clear separation of the part of screen used to select a Health Authority from the list of already configured Has above (both plain white background)
o   Health Authorities that I have already added continue to appear as options in the list.  But I can’t add them twice, so I’d expect them to disappear from the selection list once selected.
o   “Paste your URL here”.  I’d prefer “Type or Paste your URL here”
o   When I enter a 2nd URL, the previous URL I entered is offered as the default value.  I don’t consider this helpful: whatever URL I am trying to add, it is not the one I already added.

**Comment by E3V3A** [comment]
Note: 
* if you use `* this is my bullet` you will get proper bullet points here...
- [ ] You can also use `- [ ] checkbox for bullets with checkboxes`.

**Comment by StartupSam** [comment]
@diarmidmackenzie I coded that screen just to get the functionality in, with the intention that the UI would be replaced along with everything else in the v2. Looks like that functionality's not yet represented in the v2 Figma design though - have pinged the design team

**Comment by kenpugsley** [comment]
To pile on ... on iPhone the add authority by URL doesn't work because the keyboard covers up the string being entered.
![12521A8A-5945-45FB-8524-92B9A5AA8285_1_105_c](https://user-images.githubusercontent.com/4031514/80137518-05960700-8569-11ea-8668-842ba8b78ba7.jpeg)

**Comment by summetj** [comment]
A few of these have been resolved as the app has been improved. However, the following still exist:
- “Save” (floppy disk) icon used to add a URL entry. This doesn’t match any UX mechanic that I am familiar with in mobile apps for adding entries to a list.
-o Health Authorities that I have already added continue to appear as options in the list.
 (not as big of an issue, but it would be a nice improvement).
Issue #683 covers several of these as well, anybody working on this one should look at that one and visa-versa.
